### PR TITLE
v1.5.4 linked users endpoints [BAC-2638]

### PR DESCRIPTION
**Pull request by jiajames**

### CODE REVIEW

File: dydx3/modules/private.py [hunk 1]
```diff
@@ -1185,6 +1185,62 @@ def get_profile(
         '''
         return self._get('profile/private', {})
 
+    def get_user_links(
+        self,
+    ):
+        '''
+        Get Active Linked Users
+
+        :returns: UserLinks
```
Comment (eyqs) [inline dydx3/modules/private.py:1194]: Where's `UserLinks` defined?

File: dydx3/modules/private.py [hunk 2]
```diff
@@ -1185,6 +1185,62 @@ def get_profile(
...
+
+        :raises: DydxAPIError
+        '''
+        return self._get('users/links', {})
+
+    def send_link_request(
+        action,
+        address,
+        self,
```
Comment (eyqs) [inline dydx3/modules/private.py:1203]: nit: `self` first

File: dydx3/modules/private.py [hunk 3]
```diff
@@ -1185,6 +1185,62 @@ def get_profile(
...
+            },
+        )
+
+    def get_user_pending_link_requests(
+        self,
+    ):
+        '''
+        Get Pending Linked User Requests
+
+        :returns: UserLinkRequests
```
Comment (eyqs) [inline dydx3/modules/private.py:1238]: Where is `UserLinkRequests` defined?

File: dydx3/modules/private.py (hunk 2 above)
Comment (jiajames) [reply dydx3/modules/private.py]: done

File: dydx3/modules/private.py (hunk 1 above)
Comment (jiajames) [reply dydx3/modules/private.py:1194]: we typically don't define return objects in the python client, they have to look at API documentation

File: dydx3/modules/private.py (hunk 3 above)
Comment (jiajames) [reply dydx3/modules/private.py:1238]: ditto: we typically don't define return objects in the python client, they have to look at API documentation

**Review by eyqs — APPROVED** [review]
LGTM

**Pull request merged**